src/components/ShareDialog.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  Dialog,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from '@material-ui/core';
+import {
+  FacebookShareButton,
+  FacebookIcon,
+  EmailShareButton,
+  EmailIcon,
+  LinkedinShareButton,
+  LinkedinIcon,
+  RedditShareButton,
+  RedditIcon,
+  TwitterShareButton,
+  TwitterIcon,
+} from 'react-share';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    background: theme.palette.background.secondary,
+    color: 'white',
+    fontSize: '15px',
+    textAlign: 'center',
+  },
+  shareIcon: {
```

Comment:
This isn't used

Reply:
Right, initially because it did not function since I had misspelled `marginRight ` 
This is intended to give the share buttons some breathing space, it can always get removed to modified